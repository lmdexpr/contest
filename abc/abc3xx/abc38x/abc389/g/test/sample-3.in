10 433416647
