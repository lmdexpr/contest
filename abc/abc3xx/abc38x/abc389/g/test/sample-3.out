49218750 419111280 321937732 107111441 372416570 351559278 312484809 334285827 317777667 211471846 58741385 422156135 323887465 54923551 121645733 94354149 346849276 72744827 385773306 163421544 351691775 59915863 430096957 166653801 346330874 185052506 245426328 47501118 7422030 899640 79380 4536 126 0 0 0 0
